web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -40,11 +46,11 @@ consider when writing a PDEP are:
 
 ### PDEP authors
 
-Anyone can propose a PDEP, but in most cases developers of pandas itself and related
-projects are expected to author PDEPs. If you are unsure if you should be opening
-an issue or creating a PDEP, it's probably safe to start by
-[opening an issue](https://github.com/pandas-dev/pandas/issues/new/choose), which can
-be eventually moved to a PDEP.
+Anyone can propose a PDEP, but core members need to sponsor a proposal made by non-core
+contributors. To submit a PDEP as a community member, please propose the PDEP concept on
+[an issue](https://github.com/pandas-dev/pandas/issues/new/choose), and find a pandas team
+member to collaborate with. They can co-author the PDEP with you and submit it to the PDEPs
```

Comment:
@lithomas1 at https://github.com/pandas-dev/pandas/pull/51417/files#r1107953261

> Also, are PDEPs moving out of the main repo? (I'd be +1 on this)

We briefly discussed this at one of the first governance meetings, and in that sub-group there was general consensus to do that. At the time I brought that up on the pandas-dev mailing list (https://mail.python.org/pipermail/pandas-dev/2022-November/001547.html), and there were no objections to it (but also no explicit approvals ;)). I am still planning to do this.